packages/ndla-ui/src/TagSelector/Menu.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import css from '@emotion/css';
+import { colors } from '@ndla/core';
+import React from 'react';
+import { MenuProps, components } from 'react-select';
+import { StyledMenuList } from './MenuList';
+import { TagType } from './types';
+
+const menuStyle = css`
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  margin: 0;
+  overflow: hidden;
+  border-top: 1px solid ${colors.brand.tertiary};
+  min-height: 70px;
+
+  :has(${StyledMenuList}>*:only-child) {
```

Comment:
Er det en fare for at denne har for stort scope?

Reply:
Ganske sikker på at den er trygg. Hadde ikke tilgang på synlige options fra props, så måtte finne en alternativ løsning for å sjekke om menyen kun inneholder én item.